src/script_lua.c

```diff
@@ -173,6 +173,7 @@ void* luaGetFromRegistry(lua_State* lua, const char* name) {
     lua_gettable(lua, LUA_REGISTRYINDEX);
 
     if (lua_isnil(lua, -1)) {
+        lua_pop(lua, 1); /* pops the value */
```

Comment:
please re-add the text to the top comment that explains why this bug wasn't causing any real issue

Reply:
Done

Reply:
so that leak **is** reachable?
shall we back-port it to 7.0 then?

Reply:
Yes.

Reply:
spoke with Meir again, and realized i misunderstood (top comment is right though).
it is reachable, but completely harmless.
let's skip backporting it to 7.0.